src/API/Authentication.php

```diff
@@ -80,14 +80,20 @@ public function getHttpClient(): HttpClient
     public function getAuthorizationLink(
         ?array $params = null
     ): string {
-        $params = Shortcut::mergeArrays([
+        $returnUri = $this->configuration->getRedirectUri();
```

Comment:
The redirectUri could also be part of $param, right?

Reply:
Good catch, thanks! Fixed w/ a separate commit to the 8.x branch.